docs/Advanced-Usage/Debugging-and-Profiling-on-FPGA/Debugging-Hardware-Using-ILA.rst

```diff
@@ -18,6 +18,11 @@ signals directly in the Chisel source. These will be consumed by a downstream
 FIRRTL pass which wires out the annotated signals, and binds them to an
 appropriately sized ILA instance.
 
+Enabling AutoILA
+----------------
+
+To enable AutoILA, mixins `WithILATopWiringTransform` and `WithAutoILA` must be appended to the `PLATFORM_CONFIG`. These are appended by default to the `BaseF1Config`.
```

Comment:
I think you can just omit the "WithILAWiringTransform" since for the default TARGET_PROJECT, which has all of the designs people typically use, the transform is always scheduled.

Reply:
Thanks and makes sense — has been omitted.